Found no mapping for -150 (all_parameters.default_value sql_variant)

Please see below exception, I am a little confused as this error is raised after it say's Exported all_objects successfully. 


All I am trying to do is, trying to export QModel objects from the database. I see the QSources being generated but could not come to a conclusion as why this error is shown. Any help is greatly appreciated.

Thanks

```
[INFO] Exported CHECK_CONSTRAINTS successfully
[INFO] Exported COLUMN_DOMAIN_USAGE successfully
[INFO] Exported COLUMN_PRIVILEGES successfully
[INFO] Exported COLUMNS successfully
[INFO] Exported CONSTRAINT_COLUMN_USAGE successfully
[INFO] Exported CONSTRAINT_TABLE_USAGE successfully
[INFO] Exported DOMAIN_CONSTRAINTS successfully
[INFO] Exported DOMAINS successfully
[INFO] Exported KEY_COLUMN_USAGE successfully
[INFO] Exported PARAMETERS successfully
[INFO] Exported REFERENTIAL_CONSTRAINTS successfully
[INFO] Exported ROUTINE_COLUMNS successfully
[INFO] Exported ROUTINES successfully
[INFO] Exported SCHEMATA successfully
[INFO] Exported TABLE_CONSTRAINTS successfully
[INFO] Exported TABLE_PRIVILEGES successfully
[INFO] Exported TABLES successfully
[INFO] Exported VIEW_COLUMN_USAGE successfully
[INFO] Exported VIEW_TABLE_USAGE successfully
[INFO] Exported VIEWS successfully
[INFO] Exported all_columns successfully
[INFO] Exported all_objects successfully
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 39.969 s
[INFO] Finished at: 2014-12-18T15:09:25-06:00
[INFO] Final Memory: 24M/397M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal com.mysema.querydsl:querydsl-maven-plugin:3.6.0:export (generate-schema-dsl) on project ef-model: Execution generate-schema-dsl of goal com.mysema.querydsl:querydsl-maven-plugin:3.6.0:export failed: Found no mapping for -150 (all_parameters.default_value sql_variant) -> [Help 1]
org.apache.maven.lifecycle.LifecycleExecutionException: Failed to execute goal com.mysema.querydsl:querydsl-maven-plugin:3.6.0:export (generate-schema-dsl) on project ef-model: Execution generate-schema-dsl of goal com.mysema.querydsl:querydsl-maven-plugin:3.6.0:export failed: Found no mapping for -150 (all_parameters.default_value sql_variant)
	at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:224)
	at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:153)
	at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:145)
	at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject(LifecycleModuleBuilder.java:108)
	at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject(LifecycleModuleBuilder.java:76)
	at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build(SingleThreadedBuilder.java:51)
	at org.apache.maven.lifecycle.internal.LifecycleStarter.execute(LifecycleStarter.java:116)
	at org.apache.maven.DefaultMaven.doExecute(DefaultMaven.java:361)
	at org.apache.maven.DefaultMaven.execute(DefaultMaven.java:155)
	at org.apache.maven.cli.MavenCli.execute(MavenCli.java:584)
	at org.apache.maven.cli.MavenCli.doMain(MavenCli.java:213)
	at org.apache.maven.cli.MavenCli.main(MavenCli.java:157)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced(Launcher.java:289)
	at org.codehaus.plexus.classworlds.launcher.Launcher.launch(Launcher.java:229)
	at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode(Launcher.java:415)
	at org.codehaus.plexus.classworlds.launcher.Launcher.main(Launcher.java:356)
Caused by: org.apache.maven.plugin.PluginExecutionException: Execution generate-schema-dsl of goal com.mysema.querydsl:querydsl-maven-plugin:3.6.0:export failed: Found no mapping for -150 (all_parameters.default_value sql_variant)
	at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo(DefaultBuildPluginManager.java:144)
	at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:208)
	... 19 more
Caused by: java.lang.IllegalStateException: Found no mapping for -150 (all_parameters.default_value sql_variant)
	at com.mysema.query.sql.codegen.MetaDataExporter.handleColumn(MetaDataExporter.java:259)
	at com.mysema.query.sql.codegen.MetaDataExporter.handleTable(MetaDataExporter.java:334)
	at com.mysema.query.sql.codegen.MetaDataExporter.export(MetaDataExporter.java:229)
	at com.mysema.query.maven.AbstractMetaDataExportMojo.execute(AbstractMetaDataExportMojo.java:410)
	at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo(DefaultBuildPluginManager.java:133)
	... 20 more
[ERROR] 
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/PluginExecutionException
```

--- a/querydsl-sql-codegen/src/main/java/com/querydsl/sql/codegen/MetaDataExporter.java
+++ b/querydsl-sql-codegen/src/main/java/com/querydsl/sql/codegen/MetaDataExporter.java
@@ -13,7 +13,6 @@
  */
 package com.querydsl.sql.codegen;
 
-import javax.annotation.Nullable;
 import java.io.File;
 import java.io.IOException;
 import java.io.StringWriter;
@@ -22,6 +21,11 @@ import java.sql.DatabaseMetaData;
 import java.sql.ResultSet;
 import java.sql.SQLException;
 import java.util.*;
+
+import javax.annotation.Nullable;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 import com.google.common.io.Files;
 import com.mysema.codegen.CodeWriter;
@@ -34,8 +38,6 @@ import com.mysema.codegen.model.TypeCategory;
 import com.querydsl.codegen.*;
 import com.querydsl.sql.*;
 import com.querydsl.sql.codegen.support.*;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
 
 /**
  * MetadataExporter exports JDBC metadata to Querydsl query types
@@ -266,7 +268,7 @@ public class MetaDataExporter {
                 columnDigits != null ? columnDigits.intValue() : 0,
                 tableName, columnName);
         if (clazz == null) {
-            throw new IllegalStateException("Found no mapping for " + columnType + " (" + tableName + "." + columnName + " " + typeName + ")");
+            clazz = Object.class;
         }
         TypeCategory fieldType = TypeCategory.get(clazz.getName());
         if (Number.class.isAssignableFrom(clazz)) {

